src/QMCDrivers/VMC/VMCBatched.cpp

```diff
@@ -16,6 +16,7 @@
 #include "Concurrency/Info.hpp"
 #include "Utilities/RunTimeManager.h"
 #include "ParticleBase/RandomSeqGenerator.h"
+#include <boost/range/combine.hpp>
```

Comment:
Is this used somewhere?

Reply:
Good catch, the experiment with boost::combine didn't work as I liked.  std::transform for the most part accomplished what I needed.